pandas/core/ops.py

```diff
@@ -136,6 +137,62 @@ def maybe_upcast_for_op(obj):
     return obj
 
 
+class CompWrapper(object):
+    __key__ = ['list_to_array', 'validate_len',
+               'zerodim', 'inst_from_senior_cls']
+
+    def __init__(self,
+                 list_to_array=None,
```

Comment:
can you make these arg names even more explict, e.g.
``zerodim`` -> ``unwrap_zero_dim_arrays``
``validate_len`` -> ``validate_len_of_input_arrays``
``inst_from_senior_cls`` -> not really sure what this does, can you make more explicit
``list_to_array`` -> same